index.html

```diff
@@ -2,12 +2,16 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="css/main.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=PT+Serif:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600&display=swap" rel="stylesheet">
   <title>My team page</title>
 </head>
 <body>
-  <main class="container">
-    <h1 class="creative-crew-title">Hello World</h1>
-  </main>
+  <div id="root"></div>
+  <script type="module" src="index.js"></script>
 </body>
-</html>
+</html>
```

Comment:
👀 I shouldn't be pointing this at you. 
![image](https://user-images.githubusercontent.com/29681222/123025885-f561fd80-d3a0-11eb-8298-3cc427cf6783.png)